test/schematic_test.exs

```diff
@@ -16,6 +19,13 @@ defmodule SchematicTest do
   end
 
   describe "unify" do
+    property "input |> unify |> dump == input" do
+      check all({schematic, input} <- Generators.schematic_and_data()) do
+        {:ok, input} ==
+          unify(schematic, input) |> then(fn {:ok, result} -> dump(schematic, result) end)
+      end
+    end
```

Comment:
will this replace the other properties?

Reply:
most of them, i think. i'm gonna comb through today and remove the ones that are definitely overshadowed by this

Reply:
👏🏻

Reply:
replaced a bunch of existing properties that were superseded by this one. still a few remaining, but i suspect only one or two besides this are worth keeping. lmk if you feel differently